catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -719,8 +717,8 @@ private void logFailedQueryInfo(UpdateRequest updateRequest, QueryResponse query
             LOGGER.debug(
                     "While rewriting the query, did not get a metacardId corresponding to every attribute.");
             LOGGER.debug("Original Update By attribute was: {}", attributeName);
-            LOGGER.debug(
-                    "Metacards unable to get Metacard ID from are: {}", updateRequest.getUpdates()
+            LOGGER.debug("Metacards unable to get Metacard ID from are: {}",
```

Comment:
I just reformatted this, but that is a pretty odd wording. I'll fix it with the other minor change.